README.md

```diff
@@ -345,7 +345,44 @@ input encrypted amount that was removed from the sender's account.
 
 #### `amountAdded`/`amountSubtracted` (optional)
-This field is present if the `status` field is `committed` or `finalized`, and the `outcome` field is `success`. For a `TransferToPublic`, the field will be named `amountAdded` and it represents the plaintext amount that is added to the public balance of the sender. For a `TransferToEncrypted`, the field will be named `amountSubtracted` and it represents the plaintext amount that is subtracted from the public balance of the sender.
+This field is present if the `status` field is `committed` or `finalized`, and the `outcome` field is `success`, and the transaction is either `TransferToPublic` or `TransferToEncrypted`. For a `TransferToPublic`, the field will be named `amountAdded` and it represents the plaintext amount that is added to the public balance of the sender. For a `TransferToEncrypted`, the field will be named `amountSubtracted` and it represents the plaintext amount that is subtracted from the public balance of the sender.
+
+#### `moduleRef` (optional)
+This field is present if the `status` field is `committed` or `finalized`, and the `outcome` field is `success`, and the transaction is either `DeployModule` or `InitContract`. The value is either the module reference of the deployed module or the module reference from which a contract was initializad.
+
+#### `address` (optional)
+This field is present if the `status` field is `committed` or `finalized`, and the `outcome` field is `success`, and the transaction is `InitContract`. The value is the contract address of the initialized contract.
+
+#### `initName` (optional)
+This field is present if the `status` field is `committed` or `finalized`, and the `outcome` field is `success`, and the transaction is `InitContract`. The value is the name of the contract init function being called.
+
+#### `contractVersion` (optional)
+This field is present if the `status` field is `committed` or `finalized`, and the `outcome` field is `success`, and the transaction is `InitContract`. The value is the version of the contract that was initialized.
+
+#### `events` (optional)
+This field is present if the `status` field is `committed` or `finalized`, and the `outcome` field is `success`, and the transaction is `InitContract`. The value is a list of contract events, given as hex strings.
+
+#### `trace` (optional)
+This field is present if the `status` field is `committed` or `finalized`, and the `outcome` field is `success`, and the transaction is `Update`. The value is a list of elements with the following fields:
+- `"type"` - the event type. The possible values are `"updated"`, `"transferred"`, `"interrupted"` and `"resumed"`
+- if the value of the `"type"` field is `"updated"`, the following additional fields are present:
+  - `"address"` - the address of the contract that was updated,
+  - `"instigator"` - the address of the instigator of the update, i.e. source of the message, an account or contract,
```

Comment:
Explain that this can (and will) differ from the sender of the transaction in nested transactions.

Reply:
Done.